jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/frames/FlushFrame.java

```diff
@@ -0,0 +1,28 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.http2.frames;
+
+public class FlushFrame extends Frame
+{
+    public FlushFrame()
+    {
+        super(FrameType.FLUSH);
+    }
+
+    @Override
+    public String toString()
+    {
+        return String.format("%s", super.toString());
```

Comment:
Not necessary, this is equivalent to `super`.

Can we make this class `private`? Do we really need it?